actors/builtin/miner/miner_state_test.go

```diff
@@ -398,13 +398,76 @@ func TestPostSubmissionsBitfield(t *testing.T) {
 	})
 }
 
+func TestVestingFunds_AddLockedFunds(t *testing.T) {
+	t.Run("LockedFunds increases with sequential calls", func(t *testing.T) {
+		harness := constructStateHarness(t, abi.ChainEpoch(0))
+		vspec := &miner.VestSpec{
+			InitialDelay: 0,
+			VestPeriod:   1,
+			StepDuration: 1,
+			Quantization: 1,
+		}
+
+		vestStart := abi.ChainEpoch(10)
+		vestSum := abi.NewTokenAmount(100)
+
+		harness.addLockedFunds(vestStart, vestSum, vspec)
+		assert.Equal(t, vestSum, harness.s.LockedFunds)
+
+		harness.addLockedFunds(vestStart, vestSum, vspec)
+		assert.Equal(t, big.Mul(vestSum, big.NewInt(2)), harness.s.LockedFunds)
+	})
+
+}
+
+func TestVestingFundsStore(t *testing.T) {
+	harness := constructStateHarness(t, abi.ChainEpoch(0))
+	vspec := &miner.VestSpec{
```

Comment:
lets try other values here for the vesting spec in another PR? InitialDelay and VestPeriod will be much longer than StepDuration which is longer than Quantization. Thanks!

Reply:
@frrist has many more tests coming in #311, this is just a quick one to fix the bugs found so far.

Reply:
great!